Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -29,6 +29,16 @@
 
 #include <vector>
 
+#include <sstream>
+
+template <typename T>
+std::string NumberToString ( T Number )
```

Comment:
this can be replaced by ```boost::lexical_cast<std::string>(number)``` as well, we use this in other places and might be more consistent.

Also, we tend not to start with uppercase for variable names.